tailwind.config.js

```diff
@@ -2,7 +2,15 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}",],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'hoboken-blue': '#697E90',
```

Comment:
Nit: My only thing is if we are going to use these colors elsewhere besides button and in the header then I would make them less specific names.

Reply:
button color I def agree on, hoboken-blue is called that as that's the exact color they use for the hoboken branding and website